### PR TITLE
Fixing openshift-console-master.yaml to use dockerfile from ocp-build-data

### PR DESCRIPTION
**Pull request by alvaroaleman**

### CODE REVIEW

File: ci-operator/config/openshift/console/openshift-console-master.yaml [hunk 1]
```diff
@@ -9,7 +9,8 @@ build_root:
     namespace: openshift
     tag: tectonic-console-builder-v20
 images:
-- from: base
+- dockerfile_path: Dockerfile.product
```
Comment (spadgett) [inline ci-operator/config/openshift/console/openshift-console-master.yaml:12]: I wouldn't expect this to work in CI. This requires yarn offline to build console.

cc @sosiouxme

**Comment by spadgett** [comment]
/hold
Adding a hold because I believe this will break console CI.

File: ci-operator/config/openshift/console/openshift-console-master.yaml (hunk 1 above)
Comment (sosiouxme) [reply ci-operator/config/openshift/console/openshift-console-master.yaml:12]: yes, this build is very specific to the dist-git environment and i'd be hard-pressed to adapt it to work for CI as well. i think we may simply need to carry a difference here until we have cachito support for yarn in OSBS.

**Comment by alvaroaleman** [comment]
@spadgett We require you to use the same Dockerfile for both releases and CI. If there are differences, you can build switching based on the `OPENSHIFT_CI=true` env variable, which will only be present in CI

**Comment by spadgett** [comment]
Hey @jupierce, just a heads up that e2e-gcp-console is largely broken right now. We're working on it.It looks like the failures are unrelated to this PR.

**Comment by jupierce** [comment]
@spadgett Thanks. I think this migration is good otherwise -- merge whenever you can. @alvaroaleman is adding console to the exclusion list for the tooling.

**Pull request merged**